lib/MCT/Model/User.pm

```diff
@@ -45,6 +45,40 @@ sub validate {
 
 sub valid_t_shirt_sizes { qw( XS S M XL XXL ) }
 
+sub presentations {
+  my ($self, $cb) = @_;
+  #TODO add ability to only select by conference
+  #TODO select status once it exists
+
+  my $sql = <<'  SQL';
+    SELECT
+      p.id,
+      c.identifier as conference,
+      c.name as conference_name,
+      u.username as author,
+      u.name as author_name,
+      p.url_name as url_name,
+      p.title as title,
+      p.abstract as abstract
+    FROM presentations p
+    JOIN conferences c ON c.id=p.conference
+    JOIN users u ON u.id=p.author
+    WHERE p.author=?
+    ORDER BY c.created DESC, p.title
+  SQL
+
+  Mojo::IOLoop->delay(
+    sub { $self->_query($sql, $self->id, shift->begin) },
+    sub {
+      my ($delay, $err, $results) = @_;
+      die $err if $err;
+      $self->$cb(undef, [map { MCT::Model::Presentation->new(%$_, db => $self->db) } $results->hashes->each]);
```

Comment:
I originally intentionally didn't load a full object since now we are opening ourselves up for skew between this mechanism of loading presentation objects and the main class.